waterlog/src/Stylesheets/_tank.scss

```diff
@@ -0,0 +1,25 @@
+.tankComponent {
+    color: #fff;
+    display: inline-block;
+    text-align: center;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
```

Comment:
empty line between declarations and add empty line at the end of file

Reply:
done